packages/scheduler/lib/workers/scheduling/scheduling.worker.ts

```diff
@@ -58,20 +60,27 @@ export class SchedulingChild {
     private parent: MessagePort;
     private cancelled: boolean = false;
     private tickIntervalMs = 100;
+    private electionLeader: LeaderElection;
+    private nodeId = uuidv7();
 
     constructor(parent: MessagePort, db: knex.Knex) {
         if (isMainThread) {
             throw new Error('Scheduling should not be instantiated in the main thread');
         }
         this.db = db;
+        this.electionLeader = new LeaderElection({
+            db,
+            leaseTimeoutMs: 30_000,
```

Comment:
picked 30s because worse case scenario it would take less than a minute for a new leader to be elected if the previous one crashed, which I think is acceptable for our use case.
That also means that when running locally if stopping/starting the apps quickly the lease might not have expired but the scheduling worker will eventually be elected leader on the second attempt